src/elements/hx-toggle/index.js

```diff
@@ -0,0 +1,28 @@
+import { HXFormControlElement } from '../../interfaces/HXFormControlElement/index.js';
+
+import shadowMarkup from './_shadow.html';
+import shadowStyles from './_shadow.scss';
+
+/**
+ * Defines behavior for the `<hx-toggle>` element.
+ * NOTE: `<hx-toggle>` can have various default options, or it can be overriden.
+ *
+ * @extends HXFormControlElement
+ * @hideconstructor
+ * @since 0.24.0
+ */
+export class HXToggleElement extends HXFormControlElement {
+    /** @override */
+    static get is () {
+        return 'hx-toggle';
+    }
+
+    /** @override */
+    static get template () {
+        return `<style>${shadowStyles}</style>${shadowMarkup}`;
+    }
+    
+    get controlElement () {
+        return this.querySelector('input[type="checkbox"]');
+    }
```

Comment:
Please remove this method, since it's for the `<hx-toggle-control>` element.

Reply:
Changes have been done, please review.